server.js

```diff
@@ -53,16 +53,19 @@ app.get("/concentration", (req, res) => {
   );
 });
 
+//TODO delete the UNION here!
 app.get("/courses/major", (req, res) => {
   checkConnection();
   connection.query(
-    `SELECT co.subject, co.number, co.name, co.credits
+    `SELECT co.subject, co.number, co.name, co.credits, co.preReq
 	FROM major m
 	JOIN majorcategory mc ON m.idMajor = mc.majorId
 	JOIN category c ON mc.categoryId = c.idCategory
 	JOIN coursecategory cc ON c.idCategory = cc.categoryId
 	JOIN course co ON cc.courseId = co.idCourse
-	WHERE m.idMajor = ?`,
+	WHERE m.idMajor = ?
+  UNION 
+  SELECT "CS", "144", "Computer Science I", "4", "" FROM major m`,
```

Comment:
Everything after the `UNION` (including the union) should be removed.

Reply:
I left it in because we would probably want to keep it for showcasing the prereq stuff for our sprint 2 presentation to the class

Reply:
Fair point, we should leave it in then